### PR TITLE
Replace evalSymlink() with powershell command for subPath_windows.go

### PR DESCRIPTION
**Pull request by jingxu97**
In golang, evalSymlink() does not work if windows disk driver letter is
not assigned. Replace this function with a powershell command to work
around this issue.

This PR address the issue https://github.com/kubernetes/utils/issues/172

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/volume/util/subpath/subpath_windows.go [hunk 1]
```diff
@@ -43,18 +44,36 @@ func NewNSEnter(mounter mount.Interface, ne *nsenter.Nsenter, rootDir string) In
...
+// volume (represented by volumeID) to the given path. In this case, the given path is returned.
+func evalPath(path string) (linkedPath string, err error) {
+	cmd := fmt.Sprintf("Get-Item -Path %s | Select-Object -ExpandProperty Target", path)
+	output, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
+	klog.V(4).Infof("evaluate symlink from %s: %s %v", path, string(output), err)
+	if err != nil {
+		return "", err
+	}
+	linkedPath = strings.TrimSpace(string(output))
+	if strings.HasPrefix(linkedPath, "Volume") || strings.HasPrefix(linkedPath, "\\??\\Volume") {
```
Comment (ddebroy) [inline pkg/volume/util/subpath/subpath_windows.go:58]: Do you need the first condition: `strings.HasPrefix(linkedPath, "Volume")` ? You can potentially look for other Windows prefixes like `\\.\Volume` or maybe even `\\` for generic UNC path handling.

Comment (jingxu97) [reply pkg/volume/util/subpath/subpath_windows.go:58]: Because the output from powershell gives Volumexxxx as output without \\

Comment (ddebroy) [reply pkg/volume/util/subpath/subpath_windows.go:58]: Ref about various path prefixes: https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Comment (jingxu97) [reply pkg/volume/util/subpath/subpath_windows.go]: I updated the code here by adding a volume prefix function. @ddebroy PTAL.

**Comment by ddebroy** [comment]
Thanks! /lgtm

**Pull request merged**